src/components/wallet/earn/Validate/PendingMultisig.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+    <div style="word-break: break-all">
+        <div class="refresh_div">
+            <div class="refresh">
+                <Spinner v-if="loading" class="spinner"></Spinner>
+                <button v-else @click="refresh">
+                    <v-icon>mdi-refresh</v-icon>
+                </button>
+            </div>
+        </div>
+        <div class="signer_row" v-for="owner in txOwners" :key="owner.address">
+            <fa v-if="!!owner.signature" class="success_status" icon="check-circle"></fa>
+            <div v-else class="dashed_circle"></div>
+            <div>
+                <p class="body_text">{{ owner.address }}</p>
+                <p v-if="!!owner.signature" class="success_status">
+                    ({{ $t('earn.validate.pending_multisig.signed') }})
+                </p>
+                <p v-else class="pending_status">
+                    ({{ $t('earn.validate.pending_multisig.pending') }})
+                </p>
+            </div>
+
+            <v-btn
+                v-if="!Boolean(owner?.signature) && isMyWallet(owner?.address)"
+                class="button_secondary"
```

Comment:
Can we make sure that the signature and execute transaction button are in the same spot. This would also remove the logic of checking for who the wallet owner is above.
![register msig tx pending copy](https://user-images.githubusercontent.com/80516439/230106300-85c2cac9-a1ee-46c4-8f08-12dc2058feda.png)
![2_3 signed](https://user-images.githubusercontent.com/80516439/230106316-6b028270-6e9e-4235-8cbe-b9aa9e5f3049.png)

Reply:
Additionally, as we discussed, we need to check if we can bring information that is stored from Signavault, to show the tx details that the person is signing and then executing.
To show the information `node-id` and `consortiummemberaddress`.